backend/src/main/java/gov/cdc/usds/simplereport/service/PersonService.java

```diff
@@ -361,6 +367,18 @@ public Person addPatient(
     return savedPerson;
   }
 
+  /** This method associates PhoneNumbers with a new patient and ensures there are no duplicates */
+  public List<PhoneNumber> assignPhoneNumbersToPatient(Person person, List<PhoneNumber> incoming) {
+    if (incoming == null) {
+      return null;
+    }
+
+    // we don't want to allow a patient to have any duplicate phone numbers
+    List<PhoneNumber> deduplicatedPhoneNumbers = deduplicatePhoneNumbers(person, incoming);
```

Comment:
Unnecessary assignment here - you can just 
`return deduplicatePhoneNumbers(person, incoming)`
instead.

Reply:
ok, will change! I think I broke it out that way for readability but the method is so short it's not necessary

Reply:
✅